t/cobrand/bucks.t

```diff
@@ -172,6 +177,42 @@ for my $test (
     };
 }
 
+subtest 'extra CSV columns are present' => sub {
+    $mech->log_in_ok( $counciluser->email );
+
+    $mech->get_ok('/dashboard?export=1');
+
+    my @rows = $mech->content_as_csv;
+    is scalar @rows, 4, '1 (header) + 3 (reports) = 4 lines';
+    is scalar @{$rows[0]}, 21, '21 columns present';
+
+    is_deeply $rows[0],
+        [
+            'Report ID', 'Title', 'Detail', 'User Name', 'Category',
+            'Created', 'Confirmed', 'Acknowledged', 'Fixed', 'Closed',
+            'Status', 'Latitude', 'Longitude', 'Query', 'Ward',
+            'Easting', 'Northing', 'Report URL', 'Site Used',
+            'Reported As', 'Staff User',
+        ],
+        'Column headers look correct';
+
+    is $rows[1]->[20], '', 'Staff User is empty if not made on behalf of another user';
+    is $rows[2]->[20], 'counciluser@example.com', 'Staff User is correct if made on behalf of another user';
+    is $rows[3]->[20], '', 'Staff User is empty if not made on behalf of another user';
+
+    $mech->get_ok('/dashboard?export=1&updates=1');
+
+    @rows = $mech->content_as_csv;
+    is scalar @rows, 1, '1 (header) + 0 (updates)';
+    is scalar @{$rows[0]}, 9, '9 columns present';
+    is_deeply $rows[0],
+        [
+            'Report ID', 'Update ID', 'Date', 'Status', 'Problem state',
+            'Text', 'User Name', 'Reported As', 'Staff User',
+        ],
+        'Column headers look correct';
```

Comment:
This doesn't seem to test the contents of the CSV as there are no updates - worth adding some?